stopcovid/sms/send_sms.py

```diff
@@ -27,7 +27,7 @@ def _publish_send(twilio_response: Any, media_url: Optional[str] = None) -> None
                 MessageSid=twilio_response.sid,
                 To=twilio_response.to,
                 Body=twilio_response.body,
-                Status=twilio_response.status,
+                MessageStatus=twilio_response.status,
```

Comment:
fyi on the scadmin side we don't even use his field right now and just set the status to `ACCEPTED` when processing the outbound message payloads

Reply:
perhaps that should change 🤔 idk if there's any status other than ACCEPTED that can come through here

Reply:
yeah, I just now noticed that, too. is there a reason to not use this and update scadmin to accept it?

Reply:
I guess not!